cmd/etcd.go

```diff
@@ -61,138 +62,137 @@ func newShowDDLInfoCommand() *cobra.Command {
 	return m
 }
 
-// newDelOwnerCampaign return a delete owner campaign subcommand of EtcdCommand
-func newDelOwnerCampaign() *cobra.Command {
+// newDelKeyCommand returns a delete key subcommand of EtcdCommand.
+func newDelKeyCommand() *cobra.Command {
 	m := &cobra.Command{
-		Use:   "delowner [LeaseID]",
-		Short: "delete DDL Owner Campaign by LeaseID",
-		Run:   delOwnerCampaign,
+		Use:   "delkey",
+		Short: "Delete the key associated with DDL by `delkey [key]`",
+		Run:   delKeyCommandFunc,
 	}
 	return m
 }
 
-// newDelSchemaVersion return a delete schema version subcommand of EtcdCommand
-func newDelSchemaVersion() *cobra.Command {
+// newPutKeyCommand returns a put key subcommand of EtcdCommand.
+func newPutKeyCommand() *cobra.Command {
 	m := &cobra.Command{
-		Use:   "delschema",
-		Short: "delete schema version by DDLID",
-		Run:   delSchemaVersion,
+		Use:   "putkey",
+		Short: "[ONLY FOR TEST!] put a key in the path of TiDB schema versions by `putkey [key] [value]`",
+		Run:   putKeyCommandFunc,
 	}
 	return m
 }
 
 func showDDLInfoCommandFunc(cmd *cobra.Command, args []string) {
-	var rangeQueryDDLInfo = &parameter{
-		Key:      base64Encode("/tidb/ddl"),
-		RangeEnd: base64Encode("/tidb/ddm"),
-	}
-
-	reqData, err := json.Marshal(rangeQueryDDLInfo)
+	res, err := getDDLInfo()
 	if err != nil {
 		cmd.Printf("Failed to show DDLInfo: %v\n", err)
 		return
 	}
-	req, err := getRequest(cmd, rangeQueryPrefix, http.MethodPost, "application/json",
-		bytes.NewBuffer(reqData))
+	cmd.Println(res)
+}
+
+func delKeyCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Println("Only one argument!")
+		return
+	}
+
+	key := args[0]
+	ddlOwnerKeyPrefix := "/tidb/ddl/fg/owner/"
+	if !(strings.HasPrefix(key, ddlOwnerKeyPrefix) || strings.HasPrefix(key, ddlAllSchemaVersionsPrefix)) {
```

Comment:
You need to check the key must not eqult to "/tidb/ddl/fg/owner/" or ddlAllSchemaVersionsPrefix

Reply:
If we do not check it, indeed, It will not delete all the keys.

Reply:
done.